### PR TITLE
docs(API): add description to `level` constructor option

### PR DESCRIPTION
**Pull request by KoltesDigital**
This option is described in [`logger.levels`](https://github.com/pinojs/pino/blob/master/docs/api.md#loggerlevels-object) but not in its own section.

### CODE REVIEW

**Review by mcollina — APPROVED** [review]
lgtm

File: docs/api.md [hunk 1]
```diff
@@ -57,6 +57,8 @@ The name of the logger. When set adds a `name` field to every JSON line logged.
 
 Default: `'info'`
 
+The minimum level to log, i.e. lower levels won't be outputted. This reduces the load, as typically, debug and trace messages are only useful for development, quite verbose, and not needed in production.
```
Comment (mcollina) [inline docs/api.md:60]:
```suggestion
The minimum level to log: Pino will not log messages with a lower level. Setting this option reduces the load, as typically, debug and trace logs are only valid for development, and not needed in production.
```

Comment (mcollina) [reply docs/api.md:60]: cc @jsumners

Comment (KoltesDigital) [reply docs/api.md]: Updated to your suggestion.

**Pull request merged**